actions/upload_sessions.go

```diff
@@ -282,5 +282,5 @@ func (usr *UploadSessionResource) GetPaymentStatus(c buffalo.Context) error {
 }
 
 func sqlWhereForGenesisHashAndChunkIdx(genesisHash string, chunkIdx int) string {
-	return fmt.Sprintf("(genesis_hash = %s AND chunk_idx = %d)", genesisHash, chunkIdx)
+	return fmt.Sprintf("(genesis_hash = '%s' AND chunk_idx = %d)", genesisHash, chunkIdx)
```

Comment:
This was the sql error i was getting that prompted this fix:
`mysql select many: Error 1054: Unknown column '7c762a2ee5b86dc448dbed91ca75c6cede21622113f9b21e9000b4603455726a' in 'where clause'`.  
Might be some other options:  
https://gophers.slack.com/archives/C3MSAFD40/p1525467579000325

Reply:
When you log the raw query it looks like this:  
```SELECT * from data_maps WHERE (genesis_hash = c08536ccb775919edf7950a209fd5a68cc3a8c555f3209c3a89a0ecb97989ab3 AND chunk_idx = 3) OR (genesis_hash = c08536ccb775919edf7950a209fd5a68cc3a8c555f3209c3a89a0ecb97989ab3 AND chunk_idx = 2) OR (genesis_hash = c08536ccb775919edf7950a209fd5a68cc3a8c555f3209c3a89a0ecb97989ab3 AND chunk_idx = 1) OR (genesis_hash = c08536ccb775919edf7950a209fd5a68cc3a8c555f3209c3a89a0ecb97989ab3 AND chunk_idx = 0)
```

Reply:
cc @pzhao5